frame/election-provider-support/src/lib.rs

```diff
@@ -203,7 +203,7 @@ pub trait ElectionDataProvider<AccountId, BlockNumber> {
 	/// It is assumed that this function will only consume a notable amount of weight, when it
 	/// returns `Ok(_)`.
 	fn voters(
-		maybe_max_len: Option<usize>,
+		maybe_max_len: Option<u32>,
```

Comment:
I temporarily might revert this since fixing it back to usize is easier to get a clean build, you can re-revent later.